ui/appui-layout-react/src/appui-layout-react/widget/FloatingWidget.tsx

```diff
@@ -51,10 +54,13 @@ export const FloatingWidget = React.memo<FloatingWidgetProps>(function FloatingW
       maxWidth: (isSingleTab && !userSized) ? "60%" : undefined,
     };
   }, [bounds, isSingleTab, minimized, userSized]);
+  const hideFloatingWidget = React.useMemo(() => {
```

Comment:
`useMemo` is probably not needed here - not an expensive calculation?

Reply:
Rremoved